testsuite/test-init.c

```diff
@@ -144,13 +146,11 @@ static noreturn int test_remove(const struct test *t)
 
 	exit(EXIT_SUCCESS);
 }
-DEFINE_TEST(test_remove,
-	.description = "test if libkmod's remove_module returns ok",
+DEFINE_TEST(
+	test_remove, .description = "test if libkmod's remove_module returns ok",
```

Comment:
wth is clang-format doing here. I wise we had a _Pragma() or attribute we could give to this struct to convince clang-format to stop bothering. I wonder if it works if we add a `// clang-format off` before the define.

Reply:
Indeed - not ultra happy with the reshuffle. Tinkering about - neither a `// clang-format off` on the definition helps, nor does a trailing comma.

AFAICT we are left with `//  clang-format off` individual instances, files and/or just carrying on. Let me know which you prefer.